dcli/test/src/functions/ask_test.dart

```diff
@@ -58,15 +58,15 @@ void main() {
       final validator = Ask.regExp(r'^[a-zA-Z0-9_\-]+');
 
       expect(
-        () => validator.validate('!'),
+        () => validator.validate('!',null),
```

Comment:
I think the customErrorMessage should be optional, personally I don't like seeing 'null' passed to a function  and in dart it normally isn't necessary.

I also don't like method signatures that have multiple positional args that are of the same type - people tend to reverse them.

So should we move to:
`String validate(String line, {String? customErrorMessage})`

this would be backward compatible with the existing api.